test/Microsoft.AspNet.Mvc.Razor.Test/DefaultTagHelperActivatorTest.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Http.Core;
+using Microsoft.AspNet.Mvc.ModelBinding;
+using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+using Microsoft.AspNet.Routing;
+using Microsoft.Framework.DependencyInjection;
+using Moq;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.Razor.Test
+{
+    public class DefaultTagHelperActivatorTest
```

Comment:
Needs one test that verifies when multiple initializers are setup, all of them get called and one which verifies `InitializeTagHelper<FooTagHelper>` does not get used when activating `BarTagHelper`